kubernetes/loculus/values.schema.json

```diff
@@ -1117,19 +1112,15 @@
     },
     "images": {
       "type": "object",
-      "properties": {
-        "lapisSilo": {
-          "type": "string",
-          "description": "TODO",
-          "default": "ghcr.io/genspectrum/lapis-silo:0.5.2"
+      "additionalProperties": {
```

Comment:
I guess this means if I type "web-site" instead of "website" the schema won't catch it - would named properties be better? (not blocking)

Reply:
Ah yes you're right. This was a simple way of using a common schema for all values in the dict, but I think I can also do it with named keys and a defined type

Reply:
fixed.